.gitignore

```diff
@@ -1 +1,4 @@
 Gemfile.lock
+pkg
+.bundle
+
```

Comment:
please remove the empty line here + in the .ruby-version. Maybe I wasn't precise enough in my previous comment. The file ended without a `\n` after the last character.

Reply:
Fixed